lerna.json

```diff
@@ -21,7 +21,7 @@
   ],
   "command": {
     "publish": {
-      "preDistTag": "next",
```

Comment:
In general, we're leaving `preDistTag` intact (along with `preid` below). Not that it makes much/any difference here, but it will help clarify the compare diff between the `main` and `v8` branches.